app-ios/Modules/Sources/FloorMap/FloorMapFooterSegmentedButton.swift

```diff
@@ -0,0 +1,103 @@
+                            Assets.Icons.checkmark.swiftUIImage
+                        }
+                        Text("1F")
+                            .font(.system(size: 14, weight: .medium))
+                    }
+                    .foregroundStyle(AssetColors.Secondary.onSecondaryContainer.swiftUIColor)
+                }
+            }
+
+            // B1F Button
+            Button {
+                floor = .basementFirst
+            } label: {
+                HalfCapsule(
+                    foregroundColor: floor == .basementFirst ? AssetColors.Secondary.secondaryContainer.swiftUIColor : AssetColors.floorButtonBg.swiftUIColor
+                )
+                .overlay(alignment: .center) {
+                    HStack(spacing: 8) {
+                        if floor == .basementFirst {
+                            Assets.Icons.checkmark.swiftUIImage
+                        }
+                        Text("B1F")
+                            .font(.system(size: 14, weight: .medium))
+                    }
+                    .foregroundStyle(AssetColors.Secondary.onSecondaryContainer.swiftUIColor)
+                }
+            }
+        }
+    }
+
+    /// Half a capsule
+    private struct HalfCapsule: View {
+        let foregroundColor: Color
+        private let buttonWidth: CGFloat = 104
+        private let buttonHeight: CGFloat = 40
+        private let borderLineWidth: CGFloat = 1
+
+        var body: some View {
+            ZStack(alignment: .leading) {
+                Capsule()
+                    .frame(width: buttonWidth, height: buttonHeight)
+                    .overlay {
+                        Capsule()
+                            .stroke(AssetColors.outline.swiftUIColor, lineWidth: borderLineWidth)
+                            .frame(width: buttonWidth - borderLineWidth, height: buttonHeight - borderLineWidth)
+                    }
+                Rectangle()
+                    .frame(width: buttonWidth - buttonHeight / 2, height: buttonHeight)
+                    .overlay(alignment: .top) {
+                        Rectangle()
+                            .frame(width: buttonWidth - buttonHeight / 2, height: borderLineWidth)
+                            .foregroundStyle(AssetColors.outline.swiftUIColor)
+                    }
+                    .overlay(alignment: .bottom) {
+                        Rectangle()
+                            .frame(width: buttonWidth - buttonHeight / 2, height: borderLineWidth)
+                            .foregroundStyle(AssetColors.outline.swiftUIColor)
+                    }
+                    .overlay(alignment: .leading) {
+                        // Half of borderLineWidth because of boundary line
+                        Rectangle()
+                            .frame(width: borderLineWidth / 2, height: buttonHeight)
+                            .foregroundStyle(AssetColors.outline.swiftUIColor)
+                    }
```

Comment:
Because of the background color and borderColor are different, it is realized with overlay view modifier!